### PR TITLE
feat: Plain sql to model

### PR DESCRIPTION
**Pull request by lionsardesai**
## Checklist

- [x] I have run this code and it appears to resolve the stated issue
- [ ] `README.md` updated and added information about my change

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -367,30 +372,35 @@
           "group": "navigation@3"
         },
         {
-          "command": "dbtPowerUser.testCurrentModel",
+          "command": "dbtPowerUser.sqlToModel",
```
Comment (mdesmet) [inline package.json:375]: move last as this is probably least used navigation item.

File: src/commands/sqlToModel.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    if (event === undefined) {
+      return undefined;
+    }
+    const rriter = event.nodeMetaMap.values();
+    for (const rrvalue of rriter) {
+      allmodels.push(rrvalue);
+    }
+
+    const fileText = activedoc.document.getText();
+    const compiled_sql = await project.compileQuery(fileText);
```
Comment (mdesmet) [inline src/commands/sqlToModel.ts:60]: Exceptions are handled in this method but please verify this.

File: src/commands/sqlToModel.ts [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+    const retobj = await this.altimate.runModeller({
```
Comment (mdesmet) [inline src/commands/sqlToModel.ts:62]: Add exception handling

File: src/commands/sqlToModel.ts [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    const allmodels: NodeMetaData[] = [];
+    const project = this.dbtProjectContainer.findDBTProject(currentFilePath);
+    if (project === undefined) {
+      return undefined;
+    }
+    // uncomment this line in case you need to pass everything
+    // const reciter = this.eventMap.values();
+
+    // for now, restrict it to this project only.
+    // assuming user has narrowed down to a project.
```
Comment (mdesmet) [inline src/commands/sqlToModel.ts:49]: You can remove above comment as that would not work anyway.

File: src/commands/sqlToModel.ts (hunk 2 above)
Comment (lionsardesai) [reply src/commands/sqlToModel.ts:72]: double checked. exceptions caught and error messages raised.

File: src/commands/sqlToModel.ts (hunk 3 above)
Comment (lionsardesai) [reply src/commands/sqlToModel.ts]: added catch block

File: package.json (hunk 1 above)
Comment (lionsardesai) [reply package.json:375]: moved to last.

**Review by mdesmet — APPROVED** [review]

**Pull request merged**